microsoft-edge/devtools-guide-chromium/whats-new/2023/12/devtools-120.md

```diff
@@ -0,0 +1,103 @@
+---
+title: What's new in DevTools (Microsoft Edge 120)
+description: "Edge DevTools has a new user interface. Explain with Copilot in DevTools is generally available. New Recorder tool to record and replay common user flows and measure performance. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 12/07/2023
+---
+# What's New in DevTools (Microsoft Edge 120)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## DevTools has a new user interface
+
+<!-- Subtitle: Introducing the all-new Edge DevTools user interface.  Check out our blog post to learn about the new Activity Bar, Quick View, and how to customize your developer tools in the redesigned DevTools UI. -->
+<!-- todo: the above is from Welcome 119 -->
```

Comment:
@leahmsft @zoherghadyali - confirm Subtitle for What's New 120 entry "DevTools has a new user interface".